.github/workflows/maven.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a Java project with Maven
```

Comment:
Can you extract the migration to gh-actions in another PR please?

Reply:
I think this is because of 'merge vs rebase', most probably the whole PR should be rebased on the current v1.x branch.

Reply:
+1 for rebase
it will help to keep clean the commit history

Reply:
rebased with branch v1.x